x-pack/legacy/plugins/reporting/server/browsers/create_browser_driver_factory.ts

```diff
@@ -12,8 +12,11 @@ import { installBrowser } from './install';
 import { LevelLogger } from '../lib/level_logger';
 import { KbnServer } from '../../types';
 import { PLUGIN_ID } from '../../common/constants';
+import { HeadlessChromiumDriverFactory } from './chromium/driver_factory';
```

Comment:
Unrelated: I've been thinking we might want to reuse this factory, potentially, for scheduled reports. Not sure about your thoughts on that.

Reply:
++ this is one of the guts that we can reuse, as it's not specific to scheduling